src/shared/contexts/editor/index.tsx

```diff
@@ -116,6 +141,10 @@ export const EditorProvider: FC = ({children}) => {
           ? getFluxExample(rawFn as FluxFunction)
           : (rawFn as FluxFunction)
 
+      if (isFlagEnabled('injectionViaLsp')) {
+        return injectViaLsp(ExecuteCommand.InjectFunction, fn)
+      }
```

Comment:
Since this EditorContext is used everywhere we have a monaco-editor in WRITE mode, we only need to update the injection points within this context.

Here is the function injection.
We have another EditorContext method for injecting variables.

Reply:
Note that the issue ticket for function injection (#4658) is about doing LSP function injection across the application. This is why the conditional here is impacting everywhere across the UI application, once the feature flag is on.

Whereas the schema injection (and further use of `EditorContext.injectViaLsp()` will be selectively used in the new QX DataExplorer. This matches the requirements for that part of the product.